dev-libs/botan/botan-2.0.1.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A cryptography library written in C++"
+DESCRIPTION="This is the best option for crypthography in C++"
+HOMEPAGE="https://botan.randombit.net/"
+COPYRIGHT="2000-2017 botan"
```

Comment:
Why would we reassing the copyright like this? If they say "1999-2013,2014,2015,2016,2017 The Botan Authors", then we should just use that. Reassigning to a single author is not correct.